rest-server/bin/entry.js

```diff
@@ -0,0 +1,17 @@
+require('babel-register');
```

Comment:
I was under the impression that by requiring babel-register I'd be enabling further files required by node which use es6 (i.e, import statement) to be transformed by Babel. I will look into this in further detail to better understand alternative approaches.

Reply:
Im not convinced you need to use Babel.  For which features are you transpiling for? `async / await` is available (for example) in `LTS`

Reply:
Currently if I remove my require statement for babel-register I'll run into errors with using import statements in node.

I've found this on stackoverflow as a preferred alternative to babel: 
https://nodejs.org/api/esm.html

I will try to enable the import statements through this method instead.

Reply:
Is using the native node esm .mjs extension (experimental) the best practice? 

I read about @std/esm and was able to get the server + db + tests up and running using the esm package. 

I checked out a separate branch for this as well and have pushed my work to it. This can be viewed here: 
https://github.com/pkwu/node-backend-test/tree/esm

Is this the better approach for production apps?
I will merge the esm branch to my master branch to reflect the changes if approved.